desktop/apps/artist/client/views/cv.coffee

```diff
@@ -23,24 +21,10 @@ module.exports = class CVView extends Backbone.View
         shows: @statuses.shows || @statuses.cv
     .then ({ artist }) => @trigger 'artist:cv:sync', artist
 
-  postRender: ->
-    @subViews.push rail = new ArtworkRailView
-      $el: @$(".artist-artworks-rail")
-      collection: @model.related().artworks
-      title: "Works by #{@model.get('name')}"
-      viewAllUrl: "#{@model.href()}/works"
-      imageHeight: 180
-      totalArtworksCount: @model.get('counts').artworks
-      viewAllCell: true
-
-    rail.collection.trigger 'sync'
-
   render: (artist = {}) ->
     artistMetadata = artistHelpers.artistMeta @model.toJSON()
     @$el.html template _.extend { showHelpers, artistMetadata }, artist
-    _.defer => @postRender()
-    this
-
-  remove: ->
+    if artistMetadata.length == 0
```

Comment:
This is causing a bug w/ missing CV's in production. `artistMetadata` can be blank (missing hometown/location), but they can still have a CV.